libmscore/chord.cpp

```diff
@@ -3193,16 +3250,98 @@ Element* Chord::nextElement()
 
 Element* Chord::prevElement()
       {
-      for (int v = track() - 1; staffIdx() == v/VOICES; --v) {
+      Element* e = score()->selection().element();
+      if (!e && !score()->selection().elements().isEmpty() )
+            e = score()->selection().elements().first();
+      switch(e->type()) {            
+            case ElementType::NOTE: {
+                  if(e == _notes.back()) {
```

Comment:
also no braces on "case" statements in general,
unless you need to define a local variable